src/test/java/se/diabol/jenkins/pipeline/domain/PipelineTest.java

```diff
@@ -418,7 +418,7 @@ public void testFirstUpstreamBuildFirstProjectHasJustOneUpstreamJob() throws Exc
 
         assertEquals(build.getLastBuild(), BuildUtil.getFirstUpstreamBuild(build.getLastBuild(), build));
         Pipeline pipeline = Pipeline.extractPipeline("Pipeline", build);
-        List<Pipeline> pipelines = pipeline.createPipelineLatest(1, Jenkins.getInstance());
+        List<Pipeline> pipelines = pipeline.createPipelineLatest(1, Jenkins.getInstance(), false);
```

Comment:
Please extract false to a constant declared in the beginning of the class to declare the intent of the parameter, which then can be reused throughout the test file